util/annotations/helpers.go

```diff
@@ -87,3 +92,15 @@ func hasAnnotation(o metav1.Object, annotation string) bool {
 	_, ok := annotations[annotation]
 	return ok
 }
+
+// hasTruthyAnnotationValue returns true if the object has an annotation with the specified value.
```

Comment:
```suggestion
// hasTruthyAnnotationValue returns true if the object has an annotation with a value that is not "false".
```
nit

Reply:
Also maybe hasTruthyAnnotationValue => hasTruthyAnnotation (up to you)

Reply:
I prefer `hasTruthyAnnotationValue` but I'm biased because I wrote it 😝

Reply:
Absolutely fine for me!